examples/sdk-examples/src/examples/attributeFilter/AttributeParentChildFilterExample.tsx

```diff
@@ -1,23 +1,27 @@
 // (C) 2007-2020 GoodData Corporation
 import React, { useState } from "react";
-import { ErrorComponent, OnLoadingChanged, OnError } from "@gooddata/sdk-ui";
+import { OnError, OnLoadingChanged } from "@gooddata/sdk-ui";
 import { AttributeFilter } from "@gooddata/sdk-ui-filters";
 import { BarChart } from "@gooddata/sdk-ui-charts";
 import {
-    IAttributeFilter,
-    newPositiveAttributeFilter,
     attributeDisplayFormRef,
+    IAttributeFilter,
     idRef,
+    newPositiveAttributeFilter,
 } from "@gooddata/sdk-model";
 import { Ldm, LdmExt } from "../../ldm";
 
 export const AttributeParentChildFilterExample: React.FC = () => {
-    const [filter, setFilter] = useState<IAttributeFilter>();
-    const [parentFilter, setParentFilter] = useState<IAttributeFilter>();
-    const [error, setError] = useState<any>();
+    const [filter, setFilter] = useState<IAttributeFilter>(
+        newPositiveAttributeFilter(attributeDisplayFormRef(Ldm.LocationCity), { uris: [] }),
```

Comment:
This and the next one should be `newNegativeFilter`s, positive filters with no selection are invalid.